css/githubBtn.css

```diff
@@ -1,93 +1,86 @@
 .github-btn {
   overflow: hidden;
 }
+
+/* Float left to align items in a row */
 .gh-btn,
 .gh-count,
 .gh-ico {
   float: left;
 }
+
+/* Common button and count styles */
 .gh-btn,
 .gh-count {
-  padding: 0px 5px 0px 5px;
+  padding: 0 5px;
   color: #333;
   text-decoration: none;
   white-space: nowrap;
   cursor: pointer;
   border-radius: 3px;
 }
+
+/* Button styling with gradient background for cross-browser compatibility */
 .gh-btn {
-  background-color: #eee;
-  background-image: -webkit-gradient(
-    linear,
-    left top,
-    left bottom,
-    color-stop(0, #fcfcfc),
-    to(#eee)
-  );
-  background-image: linear-gradient(to bottom, #fcfcfc 0, #eee 100%);
+  background-color: #eee; /* Fallback color if gradients are not supported */
+  background-image: -webkit-gradient(linear, left top, left bottom, from(#fcfcfc), to(#eee)); /* For old WebKit */
```

Comment:
Yes, I think we should use consistent technology regardless of new or old. Since the purpose of this PR is to add a feature and not to modernize the CSS, I think it would be best to stick with the older way of doing it.